app/tests/src/test_ingest_ca_public_charge.py

```diff
@@ -9,16 +9,39 @@
+            },
+            {
+                "url": "https://keepyourbenefits.org/en/ca/resources",
+                "title": "CALIFORNIA - FAQ and Resources: What is Public Charge",
+                "markdown": "\n#### FAQs and Resources\n\nWhat is Public Charge and what benefits are included? \n\nFind answers and information sources to these and other public charge questions\n\n[Understanding Public Charge](https://keepyourbenefits.org/en/ca/resources/public-charge)\n#### Frequently Asked Questions\n\n\r\n Learn more before making decisions about public benefits for you and your family. Here are a few Frequently Asked Questions to get you started:\r\n\n**Q. What are public benefits?**\n\n\r\n A. Public benefits are government benefits like food, cash, housing, and medical assistance for people with low or no income. Examples include CalFresh/SNAP (food stamps), CalWORKs/TANF, Public Housing, Section 8, and Medi-Cal/Medicaid.",
+                "main_content": "\n#### FAQs and Resources\n\nWhat is Public Charge and what benefits are included? \n\nFind answers and information sources to these and other public charge questions\n\n[Understanding Public Charge](https://keepyourbenefits.org/en/ca/resources/public-charge)\n#### Frequently Asked Questions\n\n\r\n Learn more before making decisions about public benefits for you and your family. Here are a few Frequently Asked Questions to get you started:\r\n\n**Q. What are public benefits?**\n\n\r\n A. Public benefits are government benefits like food, cash, housing, and medical assistance for people with low or no income. Examples include CalFresh/SNAP (food stamps), CalWORKs/TANF, Public Housing, Section 8, and Medi-Cal/Medicaid.",
+            },
+            {
+                "url": "https://keepyourbenefits.org/en/ca/updates",
+                "title": "CALIFORNIA - News Updates (English)",
+                "markdown": "\n##### Nov 12, 2024: Important Update for Immigrant Families\n\n\r\nDespite the recent election, no immigration or public benefits rules have changed or are likely to change before January 20, 2025. We are closely monitoring any policy changes and will keep this page current with the latest information.",
+                "main_content": "\n##### Nov 12, 2024: Important Update for Immigrant Families\n\n\r\nDespite the recent election, no immigration or public benefits rules have changed or are likely to change before January 20, 2025. We are closely monitoring any policy changes and will keep this page current with the latest information.",
```

Comment:
FYI: I noticed there are `\r` in your text such as `\n\n\r\n`. They caused issues at one point when scraping DPSS.